_includes/v20.1/known-limitations/cdc.md

```diff
@@ -1,3 +1,7 @@
+{{site.data.alerts.callout_info}}
+Change data capture was originally built with one case study in mind. It is currently undergoing continued development and testing. Please [file a Github issue](file-an-issue.html) with us to request a feature or if you identify a bug.
```

Comment:
After talking with SE seems like this is not necessary--the limitations kind of are themselves explanatory of the state of the product. In general, do we have an "if you encounter an issue put it here" link in our docs?